apps/web/screens/LifeEventPage/LifeEventPage.tsx

```diff
@@ -0,0 +1,344 @@
+import React, { useMemo, useRef } from 'react'
+import { Screen } from '@island.is/web/types'
+import { CustomNextError } from '@island.is/web/units/errors'
+import slugify from '@sindresorhus/slugify'
+import NextLink from 'next/link'
+import {
+  SectionWithImage,
+  Slice as SliceType,
+} from '@island.is/island-ui/contentful'
+import {
+  GridRow,
+  GridColumn,
+  Breadcrumbs,
+  Text,
+  Box,
+  GridContainer,
+  BreadCrumbItem,
+  Inline,
+  Tag,
+  Link,
+} from '@island.is/island-ui/core'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import {
+  AnchorNavigation,
+  BackgroundImage,
+  Form,
+  HeadWithSocialSharing,
+  Sticky,
+  WatsonChatPanel,
+} from '@island.is/web/components'
+import {
+  GET_LIFE_EVENT_QUERY,
+  GET_NAMESPACE_QUERY,
+} from '@island.is/web/screens/queries'
+import {
+  GetLifeEventQuery,
+  GetNamespaceQuery,
+  QueryGetLifeEventPageArgs,
+  QueryGetNamespaceArgs,
+} from '@island.is/web/graphql/schema'
+import { LinkType, useLinkResolver, useNamespace } from '@island.is/web/hooks'
+import useContentfulId from '@island.is/web/hooks/useContentfulId'
+import { useRouter } from 'next/router'
+import { Locale } from 'locale'
+import { useLocalLinkTypeResolver } from '@island.is/web/hooks/useLocalLinkTypeResolver'
+import { webRichText } from '@island.is/web/utils/richText'
+import { useI18n } from '@island.is/web/i18n'
+import { Webreader } from '@island.is/web/components'
+import { watsonConfig } from '../AnchorPage/config'
+import { parseAsBoolean } from 'next-usequerystate'
+import { createNavigation } from '@island.is/web/utils/navigation'
+
+interface LifeEventPageProps {
+  lifeEvent: GetLifeEventQuery['getLifeEventPage']
+  newLayout?: boolean
+  namespace: GetNamespaceQuery['getNamespace']
+  locale: Locale
+}
+
+export const LifeEventPage: Screen<LifeEventPageProps> = ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore make web strict
+  lifeEvent,
+  newLayout = false,
+  namespace,
+  locale,
+}) => {
+  useContentfulId(lifeEvent?.id)
+  useLocalLinkTypeResolver()
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore make web strict
+  const n = useNamespace(namespace)
```

Comment:
wondering if this variable should have a more descriptive name

Reply:
It's very much good practice to not use abbreviated var naming however, in this case, this is used everywhere in the web app as such. I believe it's best to be consistent with the rest of the codebase. This is likely left over remnants of legacy `t()` type of translations that was prevalent some years ago in quite a few web technology languages. This could be a good issue to raise for a refactor week.